types/V1.mli

```diff
@@ -102,6 +102,46 @@ module type CLOCK = sig
   (** Convert a time in seconds, as returned by {!time}, into a date
       and a time. Assumes UTC (Coordinated Universal Time), also known
       as GMT. *)
+
+end
+
+(** {1 Posix clock}
+
+    Clock returning time since the unix epoch. Subject to adjustment by eg. NTP.
+    Compatible with the Ptime library. *)
```

Comment:
Removed the line mentioning Ptime, however I think we should highlight the existence of the Ptime library as a convenient means of working with the timestamp representation that PCLOCK provides.